src/analysis/analysis.py

```diff
@@ -1,16 +1,30 @@
+import os
+
 from pyproj import Geod
 
 from config import ProgramConfig
 
 
-def write_danger_zone_data_simwrapper_csv(
-    program_conf: ProgramConfig, stats: dict[str, int], filepath: str
+def write_analysis_data_simwrapper(
+    program_conf: ProgramConfig, stats: dict[str, int], output_dir: str
```

Comment:
`output_dir` has type `str` here, but we've actually been passing a `Path` to the function so far. Just happens to work either way because `os.path.join` seemingly can handle `Path`s as well